src/main/java/delete_event_use_case/DeleteEventInteractor.java

```diff
@@ -0,0 +1,35 @@
+package delete_event_use_case;
+
+/**
+ * Delete Event Interactor class.
+ * @author : Khizer Ahmad
+ * @created: November 19th 2022
+ * @since : 2022-11-21, Mon.
+ **/
```

Comment:
Please move the documentation above the package to maintain consistency throughout everyone's files.